tests/system/aiplatform/test_e2e_tabular.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import os
+import uuid
+from tests.system.aiplatform.e2e_base import TestEndToEnd
+from urllib import request
+
+from google.cloud import aiplatform
+
+
+_PROJECT = os.getenv("BUILD_SPECIFIC_GCLOUD_PROJECT")
```

Comment:
`_PROJECT` and `_LOCATION` can be retrieved from the e2e_base module to promote reuse.